datasafe-long-run-tests/datasafe-business-tests-random-actions/src/test/java/de/adorsys/datasafe/business/impl/e2e/randomactions/framework/BaseRandomActions.java

```diff
@@ -181,6 +227,7 @@ private void executeNextAction(
 
         if (null != operation) {
             executeOperation(executor, executorService, blockedExecIds, exceptions, threadId, operation);
+            log.info(operation.toString());
```

Comment:
Logger supports stream interpolation, proper solution would be 
`log.info("Operation: {}', operation)`

Reply:
updated